blackjack/console.py

```diff
@@ -0,0 +1,105 @@
+from core import *
+
+
+def show_cards(state: GameState):
+    print("")
+    print(f'Player 1: {render_hand(state.player_hand)}')
+    print(f'The House: {render_dealer(state.dealer_hand)}\n')
+
+
+def display_winner(winner: Win, state: GameState):
+    player_total = hand_total(state.player_hand)
+    house_total = hand_total(state.dealer_hand)
+    if winner is Win.Push:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total} \n PUSH!')
+        print("")
+    if winner is Win.Player_bj:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'BLACKJACK! {render_hand(state.player_hand)}= {player_total} \nYOU WIN!')
+        print("")
+    if winner is Win.House_bj:
+        print("")
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total}')
+        print(f'BLACKJACK! {render_hand(state.dealer_hand)}= {house_total} \nHouse wins!')
+        print("")
+    if winner is Win.Player_bust:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'You Bust! {render_hand(state.player_hand)}= {player_total} \nHouse Wins!')
+        print("")
+    if winner is Win.House_bust:
+        print("")
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total}')
+        print(f'House Busts: {render_hand(state.dealer_hand)}= {house_total} \nYou Win!')
+        print("")
+    if winner is Win.House_win:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total} \n House Wins!')
+        print("")
+    if winner is Win.Player_win:
+        print("")
+        print(f'The House: {render_hand(state.dealer_hand)}= {house_total}')
+        print(f'Player 1: {render_hand(state.player_hand)}= {player_total} \n YOU Win!')
+        print("")
```

Comment:
What happens if you miss a case?

Reply:
I'm not sure what you mean.

Reply:
What if a new `Win.SomethingDifferent` enum value is added and your `if` statements don't account for it?